lms/views/api/canvas/authorize.py

```diff
@@ -89,6 +89,16 @@ def oauth2_redirect(request):
     return {}
 
 
+# Stripped down version of `lms.resources._js_config.JSConfig` implementing
+# the interface used by the `base.html.jinja2` template.
+class JSConfig:
+    def __init__(self, config):
+        self.config = config
+
+    def asdict(self):
+        return self.config
```

Comment:
The `lms.resources._js_config.JSConfig` class is heavily geared towards the LTI launch scenarios and sets a lot of properties that would not be relevant if we were re-using `context.js_config` as a way for the Python backend app to pass configuration to the frontend in non-LTI launch contexts. If we are going to do this, I could use input on how best to make `context.js_config` more general.